### ISSUE TITLE
[Bug] Move window to a space in another display doesn't work the latest release

### ISSUE THREAD

**Issue by bethandtownes** [koekeishiya/yabai#2314]
When I tried to move window between difference spaces in the same display, things works well. However, when I tried to move window from a space in display 1 to a space in display 2, the window disappears on both spaces. When run the query after the move, the information correctly says the window is on the new space in the other display however i cannot find the window in the other space, "Show All Windows" doesn't bring in up either.

I didn't disable SIP.

**Comment by keithlee5** [comment]
I had the same issue and fixed it by updating my yabai version. when I ran:
```
yabai -m window --space 3
```

The window disappears, but it did not reappear in space 3 - version `yabai-v7.1.0`, mac Sonoma 14.5.

This was fixed for me when I updated my yabai version by running `brew upgrade` and restarted - make sure your yabai version is at least `yabai-v7.1.1`.

**Comment by bethandtownes** [comment]
my yabai version is 7.1.1 
![Screenshot 2024-06-04 at 9 03 30 PM](https://github.com/koekeishiya/yabai/assets/22034141/b3dc56dc-2552-4e8f-8715-714d897f964c)

**Comment by torgeir** [comment]
Upgrading to yabai-v7.1.1 works for me too.

**Comment by AhogeK** [comment]
The same issue only occurs when the window is in a floating state, and the version is also 7.1.1.

**Comment by Fisher-Wang** [comment]
I have the same issue and the yabai version is also 7.1.1

**Comment by bethandtownes** [comment]
> The same issue only occurs when the window is in a floating state, and the version is also 7.1.1.

Yeah, I can confirm that the window I am trying to move is also in floating state as well.

**Comment by cristianoliveira** [comment]
Confirming the issue with the latest version of macOs:
```
ProductName:            macOS
ProductVersion:         14.5
BuildVersion:           23F79
```

Upgrading to 7.1.1 fixed the problem. For those using nix and nix-darwin upgrade the nixpackages to `nixos-24.05`

**Comment by Fisher-Wang** [comment]
I find a workaround, by using the author's skhd config:
```
# float / unfloat window and restore position
alt - t : yabai -m window --toggle float --grid 4:4:1:1:2:2
```

when you move a window to a new space and it doesn't appear, simple press `opt - t` twice and everything works well

**Comment by nicolaric-akenza** [comment]
i have the same issue (on 7.1.5). did someone find a workaround?

**Comment by Fisher-Wang** [comment]
> i have the same issue (on 7.1.5). did someone find a workaround?

Try this

> I find a workaround, by using the author's skhd config:
> 
> ```
> # float / unfloat window and restore position
> alt - t : yabai -m window --toggle float --grid 4:4:1:1:2:2
> ```
> 
> when you move a window to a new space and it doesn't appear, simply press `opt - t` twice and everything works well

**Comment by jjmonsalveg** [comment]
Hello for me this persist:

```text
OS: macOS Sequoia 15.3.2 arm64
yabai-v7.1.11
skhd-v0.3.9
```

I tried 
```text
# float / unfloat window and restore position
alt - t : yabai -m window --toggle float --grid 4:4:1:1:2:2
```

didn't work. What other workaround other folks tried?

**Comment by koekeishiya** [comment]
This was caused by the workaround implemented when Apple added restrictions to their API. They patched the workaround as well in the following macOS update, so this bug is now irrelevant, as the workaround no longer works at all.

If you want to move windows between spaces you need to disable SIP at this point.
